backend/api/tests/common.py

```diff
@@ -41,14 +42,13 @@ def __init__(
 
     def create_user(self):
         user, created = User.objects.get_or_create(username=self.username)
+
         if created:
             user.set_password(self.password)
             user.save()
-            # for testing purpose most authentication are done without channel allowing to mock passing channel in
-            # header, this check is necessary to not break previous tests but irl a user cannot be created
-            # without a channel
             if self.channel:
                 UserChannel.objects.create(user=user, channel_name=self.channel, role=self.role)
```

Comment:
This doesn't need saving? 🤔  (as in `user.save()` just before)

Reply:
that is a good question, during coding I put the user.save() before the UserChannel, but then Thibault asked me why, that when I notice I does not change anything.
My theory is that when you create a UserChannel it takes the user as an argument, so it must change the value in the User table and save it then